packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -406,6 +406,35 @@ void main() {
     expect(find.text('initialValue'), findsOneWidget);
   });
 
+  // Regression test for https://github.com/flutter/flutter/issues/68997.
+  testWidgets('reset calls setState only once', (WidgetTester tester) async {
+    int builds = 0;
```

Comment:
I thought this shouldn't fail even with the additional `setState`? The validator should only be run once for each frame (in `build`), no?